client/src/components/Workflow/StoredWorkflowInvocations.vue

```diff
@@ -1,6 +1,6 @@
 <template>
     <CurrentUser v-slot="{ user }">
-        <StoredWorkflowDetailsProvider v-slot="{ result: item, loading }" :stored-workflow-id="storedWorkflowId">
+        <StoredWorkflowDetailsProvider v-slot="{ result: item, loading }" :storedWorkflowId="storedWorkflowId">
```

Comment:
This might trigger lint warnings.

Reply:
I guess this is the use of ``this.$attrs`` in SingleQueryProvider. Should we do something to camel case those automatically?

The other uses are even worse than either of these options right (using camelCase in the template or auto-converting attributes)?

```
<JobProblemProvider v-slot="{ result: jobProblems }" :jobid="dataset.creating_job" @error="onError">
```

Reply:
Is this change part of the fix ? https://v2.vuejs.org/v2/style-guide/?redirect=true#Prop-name-casing-strongly-recommended seems to suggest that `:stored-workflow-id` is equivalent to `:storedWorkflowId`, but that `:stored-workflow-id` is recommended.

Reply:
It would be equivalent but we're using ``$attrs`` directly as a function arguments, we're not using them as reactive properties. This is the main issue IMO - the other part of this with the ``v-if`` is essentially purely cosmetic to prevent console errors.

Reply:
I see, that makes sense. I thought they were already automatically camel cased.

Reply:
For generic store providers the attributes are converted to camel case fyi, should maybe place this function in utils and reuse it in the future: https://github.com/galaxyproject/galaxy/blob/dev/client/src/components/providers/storeProviders.js#L187